org.eclipse.scanning.test/src/org/eclipse/scanning/test/points/AbstractGeneratorTest.java

```diff
@@ -31,11 +31,9 @@
 
 	@Before
 	public void before() throws Exception {
-
 		service = new PointGeneratorService();
 	}
 
-
 	public void checkWrtCompound(Object model, IROI roi, int size) throws Exception {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)